.editorconfig

```diff
@@ -13,6 +13,6 @@ trim_trailing_whitespace = true
 [*.bat]
 end_of_line = crlf
 
-[*.yml]
+[*.{yml,xml,json,ctp,css}]
```

Comment:
sorry :flushed: I just read the tab/spaces indentation discussion... Last time I propose someting related :smile: